nupic/frameworks/opf/metrics.py

```diff
@@ -748,48 +740,6 @@ class CustomErrorMetric(MetricsIface):
   """
     Custom Error Metric class that handles user defined error metrics
   """
-  class CircularBuffer():
```

Comment:
Please keep unrelated refactoring in separate PRs to make reviews simpler. Thanks!